wiki/forms.py

```diff
@@ -220,6 +222,10 @@ def clean_slug(self):
         slug = self.cleaned_data['slug']
         if slug.startswith("_"):
             raise forms.ValidationError(_(u'A slug may not begin with an underscore.'))
+
+        is_numeric = bool(re.match('^[0-9]+$', slug))
```

Comment:
Calling the `bool` function here is clearly an unnecessary step, which is not necessary at all. `if` instruction does not need the `bool`.

Reply:
If I see and understand everything correctly, then you don't need a regular expression either.
Need to do this
```
if slug.isdigit():
    raise forms.ValidationError(_(u'Slug name cannot contain only digits'))
```

Reply:
Thanks a lot for this comment